client/clientservice/src/main.cpp

```diff
@@ -51,9 +58,8 @@ po::variables_map parseCmdLine(int argc, char** argv) {
 }
 
 int main(int argc, char** argv) {
-  // TODO: Use config file and watch thread for logger
   auto logger = logging::getLogger("concord.client.clientservice.main");
-
+  LOG_CONFIGURE_AND_WATCH(GetLog4CplusConfigLocation(), kLogConfigRefreshIntervalInMs);
```

Comment:
We should configure the logger *before* we get it.

Reply:
Fixed.